system/jlib/jstring.hpp

```diff
@@ -554,6 +554,7 @@ inline StringBuffer& operator << (StringBuffer& s, const TValue& value)
 
 extern jlib_decl void decodeCppEscapeSequence(StringBuffer & out, const char * in, bool errorIfInvalid);
 extern jlib_decl bool strToBool(const char * text);
+inline const char *boolToStr(bool b) { return b ? "true" : "false"; }
```

Comment:
@ghalliday - auxiliary util. method addition, could put in separate JIRA/PR if you want..